python/sbp/sbp2json.py

```diff
@@ -202,16 +103,15 @@ def sbp_main(args):
                 consumed = 1
             else:
                 try:
-                    m = msg_nojit.SBP.unpack(b)
-                    m = dispatch_nojit(m)
-                    dump(args, m)
+                    m = sbp.msg.SBP.unpack(b)
+                    if not include or m.msg_type in include:
+                        m = sbp.table.dispatch(m)
+                        dump(args, m)
                     consumed = header_len + m.length + 2
-                except StreamError:
-                    # Framing failed, find next preamble
+                except (UnpackError, StreamError):
+                    break
```

Comment:
shouldn't there be a `consumed = 0` before the break??

Reply:
Scratch that, I can see that it is breaking out of the loop so `consumed` is not used